### DISCUSSION TITLE
analysisSets, dataset item in condition might not be needed

### DISCUSSION THREAD

**Discussion by dostiep** [cdisc-org/analysis-results-standard#141] [Ideas]
If I get it right, the goal of creating analysisSets elements is to reprensents all possible population set that can exist for a given set of outputs. I then wonder why we would need the "dataset" item in the "condition" element as it would mean creating as many analysisSets elements as there are analysis datasets times population flags.

**Comment by ASL-rmarshall** [comment]
This is a good point that will need to be addressed.

I think the issue is that the model currently defines too close a link between defined analysis set and one if its representations (i.e., as a dataset variable flag).  Really, there is only one defined set of subjects, but the corresponding variable flag might be represented in multiple datasets for convenience.  We do not expect to create multiple duplicate population definitions, so the model should accommodate the use of the same population flag in multiple datasets (e.g. either by allowing multiple dataset variable references, by referencing a variable definition that's independent of a dataset - similar to ItemDef instead of ItemRef in Define-XML - or in some other way).

**Comment by ASL-rmarshall** [comment]
After thinking about this a bit more, I don't think it's appropriate to reference any other dataset apart from the subject-level dataset (e.g., ADSL) in the definition of the analysis set.  An analysis set is a defined set **of subjects** so the "where clause" included in the analysis set definition is there to allow identification of a set of subjects. Although the population flag variables may be copied from ADSL into other analysis datasets, these variables are not performing the same role in subject-level vs analysis datasets:
- In the subject-level dataset, population flag variables identify subjects who are part of the population. It is therefore appropriate to include these are part of the definition of the analysis set: "if you want to know which subjects are in this analysis set, you can find them by looking for the flagged subjects in the subject-level dataset".
- In analysis datasets, population flag variables identify data records as belonging to subjects who are in that particular population.  You won't necessarily get the full set of subjects in the population if you look for flagged records in the analysis dataset, so analysis dataset population flag variables should not be referenced in the definition of the analysis set.

**Comment by dostiep** [comment]
I'm not sure I agree with the above statement, especially if we intend to use the model for automation purposes. It might cause issues if the variable representing the population to select in an AnalysisSet ID is not available in the analysis dataset of interest. 

I have the same concern when it comes to AnalysisGroup ID where I think the Dataset attribute should rather be at DisplayID level. 

I attached a file where I have included examples of new attributes in "Display" sheet highlighted in yellow. I excluded (highlighted in red) Dataset attribute in both AnalysisSet and AnalysisGroup sheets. 


[ARM_Technical_Specs_DM_AE_VS_Examples_DRAFT_20220617_PDO.xlsx](https://github.com/cdisc-org/analysis-results-standard/files/10785282/ARM_Technical_Specs_DM_AE_VS_Examples_DRAFT_20220617_PDO.xlsx)

**Comment by drewcdisc** [comment]
Move to Discussions, as this one does not require a change in the model.

**Comment by drewcdisc** [comment]
@dostiep @bhavinbusa @ASL-rmarshall I converted issue #35 into Discussion #141
